tests/e2e/vmservice_late_binding.go

```diff
@@ -269,4 +309,563 @@ var _ bool = ginkgo.Describe("[vmsvc] VM-Service-VM-LateBinding", func() {
+			wffcStorageclass, err := client.StorageV1().StorageClasses().Get(ctx, wffsStoragePolicyName,
+				metav1.GetOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("wffsStoragePolicyName: %s wffcStorageclass: %s", wffsStoragePolicyName, wffcStorageclass)
+
+			ginkgo.By("Get WFFC stotage class of NFS datastore")
+			nfsPolicyLateBinding := storagePolicyNameForNFSDatastore + "-latebinding"
+			nfsPolicyLateBindingStorageClass, err := client.StorageV1().StorageClasses().Get(ctx,
+				nfsPolicyLateBinding, metav1.GetOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("wffsStoragePolicyName: %s wffcStorageclass: %s", nfsPolicyLateBinding,
+				nfsPolicyLateBindingStorageClass)
+
+			ginkgo.By("Get WFFC stotage class of sharedVMFS datastore")
+			sharedVMFSPolicyLateBinding := storagePolicyNameForSharedVMFSDatastore + "-latebinding"
+			sharedVMFPolicyLateBindingStorageClass, err := client.StorageV1().StorageClasses().Get(ctx,
+				sharedVMFSPolicyLateBinding, metav1.GetOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("wffsStoragePolicyName: %s wffcStorageclass: %s", sharedVMFSPolicyLateBinding,
+				sharedVMFPolicyLateBindingStorageClass)
+
+			ginkgo.By("Create a PVC")
+			pvc, err := createPVC(ctx, client, namespace, nil, "", wffcStorageclass, "")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			ginkgo.By("Create a PVC")
+			pvc_NFS, err := createPVC(ctx, client, namespace, nil, "", nfsPolicyLateBindingStorageClass, "")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			ginkgo.By("Create a PVC")
+			pvc_sharedVMFS, err := createPVC(ctx, client, namespace, nil, "", sharedVMFPolicyLateBindingStorageClass,
+				"")
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+			ginkgo.By("Creating VM bootstrap data")
+			secretName := createBootstrapSecretForVmsvcVms(ctx, client, namespace)
+
+			ginkgo.By("Create vm service vm")
+			vm1 := createVmServiceVmV4(ctx, vmopC, CreateVmOptionsV4{
+				Namespace:          namespace,
+				VmClass:            vmClass,
+				VMI:                vmi,
+				StorageClassName:   wffsStoragePolicyName,
+				PVCs:               []*v1.PersistentVolumeClaim{pvc, pvc_NFS, pvc_sharedVMFS},
+				SecretName:         secretName,
+				WaitForReadyStatus: true,
+			})
+
+			ginkgo.By("Wait for VMs to come up and get an IP")
+			vmIp1, err := waitNgetVmsvcVmIpV4(ctx, vmopC, namespace, vm1.Name)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("vmIp1 : %s", vmIp1)
+
+			ginkgo.By("Waiting PVC to be in bound state")
+			pvs, err := fpv.WaitForPVClaimBoundPhase(ctx, client, []*v1.PersistentVolumeClaim{pvc, pvc_NFS, pvc_sharedVMFS},
+				pollTimeout)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			pv := pvs[0]
+			volHandle := pv.Spec.CSI.VolumeHandle
+			gomega.Expect(volHandle).NotTo(gomega.BeEmpty())
+
+		}
+
+		// ginkgo.By("Validate StorageQuotaDetails after creating the workloads")
```

Comment:
commented code, please remove it from all places

Reply:
All storageQuota validation we need to update the code to Bytes , Hence keeping it